src/components/Login/Login.js

```diff
@@ -143,10 +144,11 @@ const Login = ({ intl, location, dependencies: { dopplerLegacyClient, sessionMan
         <footer>
           <CaptchaLegalMessage />
           <p>
-            <FormattedHTMLMessage
-              tagName="small"
-              id="common.copyright_HTML"
+            <FormattedMessageMarkdown
+              container="small"
```

Comment:
Why the change here? `tagname` by `container`

Reply:
Because of `tagName` is an option of _FormattedMessage_ and _conatiner_ is a Parameter of _Markdown_ component (See [react-markdown options](https://github.com/rexxars/react-markdown#options)